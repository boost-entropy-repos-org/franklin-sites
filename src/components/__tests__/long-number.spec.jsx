import React from 'react';
import renderer from 'react-test-renderer';
import LongNumber from '../long-number';

describe('Long number component', () => {
  test('should render', () => {
    const component = renderer.create(<LongNumber number={1000000} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
