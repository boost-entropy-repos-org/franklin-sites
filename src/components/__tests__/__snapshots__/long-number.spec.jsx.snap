// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Long number component should render 1`] = `"1,000,000"`;
